config/default/core.entity_view_mode.media.caption_credit.yml

```diff
@@ -0,0 +1,10 @@
+uuid: 6e92deb8-5552-4048-a078-4d84c1f29c8d
+langcode: en
+status: true
+dependencies:
+  module:
+    - media
+id: media.caption_credit
```

Comment:
Where is a respective entity display config for this view mode?

Reply:
This was not my card but I'm assuming we are missing a config file that should have exported with view mode setup?

If so, I can try to track that down and get it added, it would be an entity display config yml?

Reply:
Correct. We can merge this work but I suspect none of the caption stuff will display because of the missing view config

Reply:
@pookmish I don't want to hold up the release to staging, can we go ahead and merge as is? Since this work is additive it shouldn't break anything and we'll work on getting a fix ready asap.

Reply:
that sounds good to me